src/js/utils.js

```diff
@@ -37,19 +37,6 @@ function flattenData(data) {
   return result;
 }
 
-
-// -- converts latlon to pixels at zoom level 0 (for 256x256 tile size) , inverts y coord )
-// -- source : http://build-failed.blogspot.cz/2013/02/displaying-webgl-data-on-google-maps.html
-function latLonToPixel(latitude, longitude) {
```

Comment:
I believe you'll want to preserve this function, and use it as a default so that when `latLonToPixel` is not defined, it is used.  That way the library stays backwards compatible with older usages if they were to upgrade.  Please make that change, and I think everything else here looks solid.

Reply:
I have added `latLonToPixel` back in. This restores backwards compatibility for any clients who are calling it directly. I'm not sure what you mean by "use it as a default" - it is used anywhere within this library.

But maybe `latLonToPixel` should just call `map.project` ?